samples/model-builder/create_batch_prediction_job_sample.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional, Sequence, Union
+
+from google.cloud import aiplatform
+
+#  [START aiplatform_sdk_create_batch_prediction_job_sample]
+def create_batch_prediction_job_sample(
+    project: str,
+    location: str,
+    model_resource_name: str,
+    job_display_name: str,
+    gcs_source: Union[str, Sequence[str]],
+    gcs_destination: str,
+    sync: bool = True,
+):
+    aiplatform.init(project=project, location=location)
+
+    batch_prediction_job = aiplatform.BatchPredictionJob.create(
```

Comment:
Slight preference here is to call this through `Model`.

Reply:
Good catch, swapping this